src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractValidationServiceImpl.java

```diff
@@ -178,16 +178,23 @@ private Person findPerson(String employeeId) {
         return null;
     }
     
-    @Deprecated
-    public boolean validateConcurStandardAccountingExtractDetailLine(ConcurStandardAccountingExtractDetailLine line) {
-        ConcurStandardAccountingExtractBatchReportData reportData = new ConcurStandardAccountingExtractBatchReportData();
-        return validateConcurStandardAccountingExtractDetailLine(line, reportData);
+    @Override
+    public boolean validateConcurStandardAccountingExtractDetailLine(ConcurStandardAccountingExtractDetailLine line, ConcurStandardAccountingExtractBatchReportData reportData) {
+        boolean valid = validateConcurStandardAccountingExtractDetailLineBase(line, reportData);
+        valid = validateAccountingLine(line, reportData) && valid;
+        return valid;
     }
 
     @Override
-    public boolean validateConcurStandardAccountingExtractDetailLine(ConcurStandardAccountingExtractDetailLine line, ConcurStandardAccountingExtractBatchReportData reportData) {
+    public boolean validateConcurStandardAccountingExtractDetailLineWithObjectCodeOverride(ConcurStandardAccountingExtractDetailLine line, 
+            ConcurStandardAccountingExtractBatchReportData reportData, String overriddenObjectCode, String overriddenSubObjectCode) {
+        boolean valid = validateConcurStandardAccountingExtractDetailLineBase(line, reportData);
+        valid = validateAccountingLineWithObjectCodeOverrides(line, reportData, overriddenSubObjectCode, null) && valid;
```

Comment:
Based on the new method signature, the third arg should be the object code and the fourth should be the sub-object code.